15_1_Building_the_Regtest.md

```diff
@@ -0,0 +1,24 @@
+# 15.1: Building the Regtest
+
+> **NOTE:** This is a draft in progress, so that I can get some feedback from early reviewers. It is not yet ready for learning.
+
+This document explains how to build a Regtest (Regression Test) by hand to be able to develop new applications without the need to interact with other peers and blocks.
+Bitcoin Core’s regression test mode (regtest mode) lets you instantly create a brand-new private block chain with the same basic rules as testnet—but one major difference: you choose when to create new blocks, so you have complete control over the environment.
+
+## Starting Bitcoind in Regtest Mode
+
+To start your Bitcoind (Bitcoin Daemon) in Regtest mode and create a private Blockchain, you have to use the following command:
+```
+$ bitcoind -regtest -daemon
+```
+
+## Resetting the Regtest Blockchain
+
+Regtest wallets and block chain state (chainstate) are saved in the regtest subdirectory of the Bitcoin configuration directory. 
+If you want to start a brand new Blockchain using the Regtest mode, all you have to do is delete the `regtest` folder and restart the Bitcoind.
+
```

Comment:
Always great to show the command line examples, so here it might be nice to see what that subdirectory looks like. I think I did the same thing in the early sections.